docs/reference/latency.md

```diff
@@ -0,0 +1,96 @@
+- The nonce and creation time for the new block, which are set by the miner.
+
+### Block parents and orphans
+
+If the block selected to serve as the parent of the new candidate block is superseded by the arrival of a new block on the target chain, the block production process must start over with the new parent block.
+If the block production process starts over, the new block candidate that was in processed is no longer valid for the continuity of the chian and is deemed an **orphaned** block.
+
+### Block height and payload
+
+The next step in the block production process involves two requirements that are satisfied in parallel. 
+
+- The first requirement is to determine whether the chains that are adjacent to the target chain in the chain graph have any blocks at the same height as the parent block.
+  If the adjacent chains have blocks at the same height as the parent block, those chains are deemed ready to add a new block and satisfy the braiding requirement of the Chainweb protocol.
+
+- The second requirement is to produce the block payload on the target chain.
+  Producing the payload involves querying the mempool for a set of transactions that can fit in the block, and executing those transactions on the Pact state.
+
+### Mining work
+
+After the block production requirements are met—that is, the block payload, parent block, and the adjacent parent blocks have been validated—the new block can be mined and added to the target chain.
+Miners submit requests to the `/mining/work` endpoint to receive work.
+The node selects a chain at random from the set of chains that are ready, and returns the information it has collected as an octet stream to the miner for use during mining.
+After mining is successful, the new block candidate is added to the target chain.
+
+## Single chain transactions
+
+On average, single chain transactions take approximately 45 seconds.
+However, transaction latency—defined as the time between submitting a transaction to a node `/send` endpoint and seeing its result with the `/poll` endpoint—can take much longer than 45 seconds, in particular, if there are mining delays.
+
+To give you better insight into how transactions are executed, the following example describes the workflow for a single chain transaction.
+You start a transaction by submitting a formatted request to the `/send` endpoint on a node.
+The code provided by the `/send` endpoint performs some simple initial checks that validate the transaction has a reasonably high chance of being executed successfully. 
+These initial checks ensure that you learn if a transaction is invalid as soon as possible. 
+If the transaction passes the initial checks, the `/send` process adds the transaction to the mempool for the local node. 
+The transaction is then gossiped to remote nodes, and, eventually, one of the nodes is selected to mine the next block. 
```

Comment:
```suggestion
The transaction is then gossiped to remote nodes, and, eventually, one of the nodes is selected to mine the next block, and it includes the transaction into that block. 
```

Reply:
Explaining how it relates to the transaction lifecycle, because the transaction is the subject of this discussion.